### PR TITLE
Feature/winter 21  disable auto add sessions

### PR DESCRIPTION
**Pull request by jjbennett**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
~~Place holder data is incorporated (Refer to [Case Management Sample Data document](https://quip.com/cbFcAPJF8t0z))~~
~~Base axe-core JEST test included for any new LWC (No critical violations)~~
~~CRUD/FLS is enforced in Apex~~
- [x] Update & Test permission sets to account for new features and CRUD/FLS
- [x] All modified classes are unit tested (Apex) at 100% coverage
- [x] UX approval or UX not necessary
~~PR contains draft release notes~~
- [x] Attach work item to the pull request with [Lurch](https://salesforce.quip.com/50ZRA5LEWVzH) `**Lurch:attach W-000000` or `**Lurch:add`
- [ ] All **acceptance criteria** have been met
    - [x] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [ ] QE story level testing completed

### CODE REVIEW

**Comment by jjbennett** [comment]
[W-8035750](https://gus.lightning.force.com/a07B0000008ITiWIAW) & [W-8015266](https://gus.lightning.force.com/a07B0000008IOlLIAW)

**Review by lbeller — CHANGES_REQUESTED** [review]
Please change: "When selected, records are created for each Service Session." to "When selected, a record is created for each Service Session."

File: force-app/main/default/lwc/scopedNotification/scopedNotification.js [hunk 1]
```diff
@@ -32,6 +35,14 @@ export default class ScopedNotification extends LightningElement {
         return iconName;
     }
 
+    get iconVariant() {
```
Comment (bethbrains) [inline force-app/main/default/lwc/scopedNotification/scopedNotification.js:38]: wouldn't this always return inverse for success, info, and error? is that really what we want?

File: force-app/main/default/lwc/serviceScheduleCreator/serviceScheduleCreator.html [hunk 2]
```diff
@@ -28,6 +28,27 @@
                 service-schedule-model={serviceScheduleModel}
             ></c-service-schedule-review>
         </template>
+        <c-modal header={labels.goBack} is-nested="true">
```
Comment (bethbrains) [inline force-app/main/default/lwc/serviceScheduleCreator/serviceScheduleCreator.html:31]: this drops another background and looks very odd. is the is-nested not working correctly?

File: force-app/main/default/objects/ServiceSchedule__c/fields/CreateServiceSessionRecords__c.field-meta.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
     <fullName>CreateServiceSessionRecords__c</fullName>
-    <defaultValue>false</defaultValue>
+    <defaultValue>true</defaultValue>
     <externalId>false</externalId>
     <inlineHelpText>When selected, records are created for each Service Session.</inlineHelpText>
```
Comment (bethbrains) [inline force-app/main/default/objects/ServiceSchedule__c/fields/CreateServiceSessionRecords__c.field-meta.xml:6]:
```suggestion
    <inlineHelpText>When selected, a record is created for each Service Session.</inlineHelpText>
```
vetted by Lauren

Comment (bethbrains) [reply force-app/main/default/objects/ServiceSchedule__c/fields/CreateServiceSessionRecords__c.field-meta.xml:6]: @lbeller like this, yes?

**Review by bethbrains — CHANGES_REQUESTED** [review]

File: force-app/main/default/objects/ServiceSchedule__c/fields/CreateServiceSessionRecords__c.field-meta.xml (hunk 3 above)
Comment (lbeller) [reply force-app/main/default/objects/ServiceSchedule__c/fields/CreateServiceSessionRecords__c.field-meta.xml:6]: Perfect!

File: force-app/main/default/lwc/scopedNotification/scopedNotification.js (hunk 1 above)
Comment (jjbennett) [reply force-app/main/default/lwc/scopedNotification/scopedNotification.js:38]: yes those 3 need the icon variant to be inverse anything else does not.

File: force-app/main/default/lwc/serviceScheduleCreator/serviceScheduleCreator.html (hunk 2 above)
Comment (bethbrains) [reply force-app/main/default/lwc/serviceScheduleCreator/serviceScheduleCreator.html:31]: let's make a WI to try to address this later. Not a dealbreaker now.

**Review by bethbrains — APPROVED** [review]
Great stuff, Jenny.

**Review by lbeller — APPROVED** [review]

**Review by AndersonTarren — APPROVED** [review]

**Pull request merged**